packages/api/src/controllers/asset.ts

```diff
@@ -321,17 +320,7 @@ export function getPlaybackUrl(
     (f) => f.type === "catalyst_hls_manifest"
   );
   if (catalystManifest) {
-    if (os.id !== vodCatalystObjectStoreId) {
-      return pathJoin(os.publicUrl, asset.playbackId, catalystManifest.path);
-    }
-    // TODO: Set up Catalyst playback or a CDN in front of the default object store.
     return pathJoin(os.publicUrl, asset.playbackId, catalystManifest.path);
-    // return pathJoin(
-    //   "https://playback.livepeer.monster:10443/", // TODO: Make this a cli arg
-    //   "hls",
-    //   `asset+${asset.playbackId}`,
-    //   "index.m3u8"
-    // );
```

Comment:
We cannot change any logic for using assets from the old pipeline until we migrate all of them to Catalyst. We can only stop _generating_ them for now.

Reply:
ahh.. of course

Reply:
But the result is the same from this logic, either way we just return `return pathJoin(os.publicUrl, asset.playbackId, catalystManifest.path);` so that's why I removed the if statement

Reply:
lol I'm sorry, I had just landed a 36h 🛬 trip yesterday morning, maybe I was a zombie 🧟